frontend/routes/Wifi.jsx

```diff
@@ -13,41 +16,95 @@ export class Component extends React.Component {
       last_refresh: '',
     };
     setInterval(() => {
-      exec_table('nmcli dev wifi').then(result => this.setState({ wifi_networks: result }));
+      exec_table('nmcli device wifi list').then(result => this.setState({ wifi_networks: result }));
       exec('date').then(result => this.setState({ last_refresh: result }));
     }, 1000);
   }
 
   render() {
     return (
-      <div>
-        <Table data={this.state.wifi_networks} />
+      <div style={{ padding: '40px' }}>
+        {this.state.wifi_networks.length == 0 ? (
+          <Typography variant="h2" color="textSecondary">
+            No wifi networks available
+          </Typography>
+        ) : (
+          <MaterialTable
+            columns={[
+              {
+                title: 'SSID',
+                field: 'SSID',
+                render: rowData => {
+                  if (rowData['IN-USE'] != '') {
+                    return <p style={{ color: '#ec0101' }}>{rowData.SSID}</p>;
+                  } else {
+                    return <p>{rowData.SSID}</p>;
+                  }
+                },
+                sorting: true,
+                defaultSort: 'asc',
+              },
+              {
+                title: 'SIGNAL',
+                field: 'SIGNAL',
+                type: 'numeric',
+                sorting: true,
+                defaultSort: 'asc',
+              },
+              {
+                title: 'SECURITY',
+                field: 'SECURITY',
+                sorting: false,
+                render: rowData => {
+                  return rowData['SECURITY'].split(' ').map(tag => {
+                    return <Chip key={tag} label={tag} variant="outlined" />;
+                  });
+                },
+              },
+
+              { title: 'MODE', field: 'MODE', sorting: false },
+              { title: 'CHAN', field: 'CHAN', sorting: false },
+              { title: 'RATE', field: 'RATE', sorting: false },
+            ]}
+            data={[...this.state.wifi_networks]}
```

Comment:
The `material-table` package doesn't seem to like non-constant data, for example, the sorting gets messed up, if the data is updated, and also the reordering of the column.

Reply:
Taken care in upcoming commits